.github/workflows/pipeline.yml

```diff
@@ -16,5 +22,8 @@ jobs:
       - name: Quality checks
         run: make quality
 
+      - name: Upload coverage to Codecov
```

Comment:
I have never used this service before - where do the results go? Some dashboard on some website somewhere?

Reply:
https://codecov.io/gh/IBM/argocd-vault-plugin/ I will also put a status badge in the readme